environs/config/config.go

```diff
@@ -588,6 +592,12 @@ func Validate(cfg, old *Config) error {
 		}
 	}
 
+	if v, ok := cfg.defined[BackupDirKey].(string); ok && v != "" {
+		if _, err := os.Open(v); err != nil {
```

Comment:
This can't be done here - validation is also done when config is processed on client side eg during bootstrap. We need to simple accept the value and instead validate when backup is done and return an error at that point. This can be tested by bootstrapping with --config option and specifying a bogus backup dir